course_content/notebooks/dask_intro.ipynb

```diff
@@ -734,6 +734,103 @@
    "source": [
     "print(delayed_merged_cube.compute())"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Exercise: Loading\n",
+    "\n",
+    "Consider the following code snippets. Despite looking similar, not all of these code snippets will provide parallel file loading:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "# This cell sets up the files needed by the exercise.\n",
+    "filepath = iris.sample_data_path('GloSea4')\n",
```

Comment:
@dkillick I think that this needs to be
```python
filepath = iris.sample_data_path('GloSea4', '*.pp')
```
Otherwise, it's all a bit confusing ...